src/microbe_stage/Microbe.cs

```diff
@@ -123,7 +123,6 @@ public class Microbe : RigidBody, ISpawned, IProcessable, IMicrobeAI, ISaveLoade
     [JsonProperty]
     private Color flashColour = new Color(0, 0, 0, 0);
 
-    [JsonProperty]
     private bool allOrganellesDivided;
```

Comment:
I wasn't initially super keen on this, but on further thinking, I think this is pretty elegant in the end. Though I want one change, this should have a comment along the lines of:
"True once all organelles are divided to not continuously run code that is triggered when a cell is ready to reproduce. This is not saved so that the player cell can enable the editor when loading a save where the player is ready to reproduce. If more code is added to be ran just once based on this flag, it needs to be made sure that that code re-running after loading a save is not a problem."